src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -177,6 +177,11 @@ public enum NuGetLogCode
         /// </summary>
         NU1109 = 1109,
 
+        /// <summary>
+        /// The package `packageId` is available in the Global packages folder, but the source it came from `package source URI` is not one of this solution's configured sources.
```

Comment:
```suggestion
        /// The package `packageId` is available in the Global packages folder, but the source it came from `package source URI` is not one of the configured sources.
```
It's just configured sources. Technically the configuration can be project based even though there's caveats so best to just not call out either.

Reply:
Gotcha! I've updated the string.